manual/bitdefender-usb-immunizer/bitdefender-usb-immunizer.nuspec

```diff
@@ -1,16 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
-<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
-    <id>{{PackageName}}</id>
-    <version>{{PackageVersion}}</version>
-    <title>Bitdefender USB Immunizer</title>
+    <id>bitdefender-usb-immunizer</id>
+    <version>2.0.1.901</version>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/bitdefender-usb-immunizer</packageSourceUrl>
     <owners>chocolatey</owners>
+    <title>Bitdefender USB Immunizer</title>
     <authors>Bitdefender SRL</authors>
-    <licenseUrl>http://labs.bitdefender.com/projects/usb-immunizer/overview/</licenseUrl>
     <projectUrl>http://labs.bitdefender.com/projects/usb-immunizer/overview/</projectUrl>
-    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/edba4a5849ff756e767cba86641bea97ff5721fe/icons/bitdefender-usb-immunizer.png</iconUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/9282e769979fbb235869051249a70d56517f4ff9/icons/bitdefender-usb-immunizer.png</iconUrl>
+    <copyright>What is the copyright</copyright>
```

Comment:
Why "What is the copyright"? Shouldn't the tag be omitted if copyright is unclear?